geomesa-web/geomesa-web-data/pom.xml

```diff
@@ -145,6 +145,23 @@
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-accumulo-datastore_${scala.binary.version}</artifactId>
         </dependency>
+
+        <!-- test dependencies -->
+
+        <dependency>
+            <groupId>org.specs2</groupId>
+            <artifactId>specs2_${scala.binary.version}</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.scalatra</groupId>
+            <artifactId>scalatra-specs2_2.11</artifactId>
```

Comment:
Should the 2.11 be ${scala.binary.version}?

Reply:
yeah, good catch

Reply:
fixed